src/test/java/onseo/jerseytest/MockSampleTest.java

```diff
@@ -31,25 +37,120 @@
     private SummaryService summaryService;
 
     @Before
-    public void setup()
-    {
-        when(toDoEntityPlaceholderClient.getEntity(Matchers.anyInt(), any(), any())).thenReturn(generateStubEntity());
-        summaryService = new SummaryService(toDoEntityPlaceholderClient, commentEntityPlaceholderClientMock, userEntityPlaceholderClientMock);
+    public void setup() throws ExecutionException, InterruptedException {
+        CompletableFuture<ToDoEntity> todoTask = CompletableFuture.supplyAsync(this::generateToDoStubEntity);
```

Comment:
You don't need to use multithreading here. It is already implemented in SummaryService. You need just return appropriate values